<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managemembers</class>
 <widget class="QWidget" name="managemembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>501</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddMemberButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add a Member</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>218</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search by Member ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchPushButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="DisplayAllPushButton">
      <property name="text">
       <string>Display All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>274</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Delete by Member ID</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="deleteMemberLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
